services/web/server/src/simcore_service_webserver/projects/projects_db.py

```diff
@@ -249,13 +253,15 @@ async def load_user_projects(self, user_id: int) -> List[Dict]:
         log.info("Loading projects for user %s", user_id)
         async with self.engine.acquire() as conn:
             user_groups: List[RowProxy] = await self.__load_user_groups(conn, user_id)
-            query = f"""
-    SELECT *
-    FROM projects
-    WHERE projects.type != 'TEMPLATE'
-    AND (jsonb_exists_any(projects.access_rights, array[{', '.join(f"'{group.gid}'" for group in user_groups)}])
-    OR prj_owner = {user_id})
-    """
+            query = textwrap.dedent(
+                f"""\
```

Comment:
just curious, what is the usage of `\` for?

Reply:
Avoids new-line at the beginning:
```cmd
>>> from textwrap import dedent
>>> A = dedent("""
one
   two
   """
)
>>> B = dedent("""\
one
   two
   """
)
>>> A
'\none\n   two\n'
>>> B
'one\n   two\n'
>>> 
```